sdk/mixedreality/mixedreality-authentication/samples/javascript/getToken.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Demonstrates something
```

Comment:
Suggestion:
```suggestion
 * This sample demonstrates how to retrieve a Mixed Reality STS access token using
 * a Mixed Reality account key.
```

Reply:
Doh! Good catch. I'll use the same content I'm using in the other language samples. Thanks!